src/rust/engine/src/lib.rs

```diff
@@ -51,6 +51,8 @@ use log;
 
 use tar_api;
 
+use workunit_store;
```

Comment:
I think this line (and the newline after it) can be removed after my suggestion below.